src/Resources/config/services/repository/credit_memo.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="bitbag_sylius_mollie_plugin.repository.credit_memo_repository" class="BitBag\SyliusMolliePlugin\Repository\CreditMemoRepository" public="true" decorates="sylius_refund.repository.credit_memo">
+            <argument type="service" id="sylius_refund.manager.credit_memo"/>
+            <argument type="service">
+                <service class="Doctrine\ORM\Mapping\ClassMetadata" public="false">
+                    <argument>%sylius_refund.model.credit_memo.class%</argument>
+                </service>
+            </argument>
+        </service>
+    </services>
```

Comment:
then you can add your repo which will have parent `sylius_refund.repository.credit_memo`